solidity/contracts/bridge/Bridge.sol

```diff
@@ -21,6 +21,7 @@ import {BTCUtils} from "@keep-network/bitcoin-spv-sol/contracts/BTCUtils.sol";
 import {BytesLib} from "@keep-network/bitcoin-spv-sol/contracts/BytesLib.sol";
 import {IWalletOwner as EcdsaWalletOwner} from "@keep-network/ecdsa/contracts/api/IWalletOwner.sol";
 
+import "./library/Frauds.sol";
```

Comment:
I moved the Frauds library to the same directory as Bridge in bf0f379933bd7a4f91c6f638551e86f7a7a8fbcc.
It would not make sense to keep just one library in a separate directory.
If we want to move the libraries there, we can move them all (in another PR).